### ISSUE TITLE
Error optaining

### ISSUE THREAD

**Issue by SzokeMartin** [TylerWilliamson/QuickWeather#126]
The app cant obtain info from the Open weather API

**Comment by TylerWilliamson** [comment]
Please provide more information:

1. Is your key valid? You must have a subscription to "OneCall"
1. When you go to Settings and click "Test Connection", does it turn green? You may have to cut and paste your API Key to enable "test connection"
1. Is there any additional information you can provide, such as error messages or screenshots? If you do provide screenshots, please obscure your API Key

**Comment by SzokeMartin** [comment]
> Please provide more information:
> 
> 1. Is your key valid? You must have a subscription to "OneCall"
> 1. When you go to Settings and click "Test Connection", does it turn green? You may have to cut and paste your API Key to enable "test connection"
> 1. Is there any additional information you can provide, such as error messages or screenshots? If you do provide screenshots, please obscure your API Key

Yes, My key Is valid, with a green tick, and i have the one call api.

**Comment by SzokeMartin** [comment]
![Screenshot_20240131_072519.png](https://github.com/TylerWilliamson/QuickWeather/assets/74468792/6dbd45de-585e-4634-93fb-91714b193ff7)

**Comment by TylerWilliamson** [comment]
The error is saying it could not reach the internet from the background task. The green check means the app successfully reached the API, at least when you set it up.

Do you see data in the app? With only that error message? If so, you can ensure that the "Allow background data" setting and "unrestricted battery access" settings are set to reduce the errors. You can also disable that notification channel if you really do not wish to see it

**Comment by SzokeMartin** [comment]
Thanks man! I appricate your work and help,and this app is good and simple!
